Feedback (#1)
sdgis-edu-tud/report-asa2025-groupe: report.qmd

---
title: "Urban Stream Restoration in the City of Dresden"
subtitle: "Researching the optimal locations to restore the urban streams in order to improve the biodiversity, climate adaptivity and quality of life in the city of Dresden"
author:
  - name: "Daan Schlosser"
    affiliation: "MSc Geomatics"
    role: "Project coordinator & data-analyst"
  - name: "Joost Bastiaansen"
    role: "Mapping specialist"
    affiliation: "MSc Urbanism"
  - name: "Aparnaa Chandrasekaran"
    role: "Research lead"
    affiliation: "MSc Urbanism"
  - name: "Teun van Dijk"
    affiliation: "MSc Urbanism"
    role: "Presentation & design lead"

format: html
---

| Name                   | Affiliation     | Role                               |
|------------------------|-----------------|------------------------------------|
| Daan Schlosser         | MSc Geomatics   | Project coordinator & data-analyst |
| Joost Bastiaansen      | MSc Urbanism    | Mapping specialist                 |
| Aparnaa Chandrasekaran | MSc Urbanism    | Research lead                      |
| Teun van Dijk          | MSc Urbanism    | Presentation & design lead         |

<br>

## Introduction

### Site Selection

> This project will focus on the city of Dresden. In order to provide an answer to the main research question “what areas of Dresden would stand to gain the most from the urban stream restoration project?” a division of areas has to be provided. To start, a grid with tiles of 500 by 500 metres was defined for the city of Dresden. This is filtered to only show the tiles which contain a stream to remove any unnecessary information.

Reviewer: ```suggestion
This project will focus on the city of Dresden. In order to provide an answer to the main research question “what areas of Dresden would stand to gain the most from the urban stream restoration project?” a division of areas has to be provided. To start, a grid with tiles of 500 by 500 metres was defined for the city of Dresden. This is filtered to only show the tile which contain a stream to remove any unnecessary information.
```
you made typo

![Step 1: A 500 by 500 metres grid covering Dresden](images\introduction\STEP1.png)

![Step 2: Overlay with streams](images\introduction\STEP2.png)

> ![Step 3: Remove any unecessary information](images\introduction\STEP3.png)

Reviewer: The description of the spatial unit should be under the Methods section.

## Methods

### MCDA

#### Quality of Life
![Physical Health: Vicinity to Urban Green Space](images\quality_of_life\VicinityToUrbanGreenSpace.jpeg)
![Physical Health: Vicinity to Urban Green Space](images\quality_of_life\VicinityToUrbanGreenSpace_Grid.jpeg)
![Mental Health: Vicinity to Stream](images\quality_of_life\VicinityToStream.jpeg)
![Mental Health: Vicinity to Stream](images\quality_of_life\VicinityToStream_Grid.jpeg)
![General Health: Facilities](images\quality_of_life\Facilities.jpeg)
![General Health: Facilities](images\quality_of_life\Facilities_Grid.jpeg)
![Accessibility: Local Integration](images\quality_of_life\LocalIntegration.jpeg)
![Accessibility: Local Integration](images\quality_of_life\LocalIntegration_Grid.jpeg)
![Accessibility: Appearance](images\quality_of_life\Covered.jpeg)

> ![Accessibility: Appearance](images\quality_of_life\Covered_Grid.jpeg)

Reviewer: All these should be in the results section. The methods section is meant to show _how_ the analysis was carried out.

### Typology construction

> ## Results

Reviewer: Have you included here all results from the midterm?

Reviewer: Assuming that you did, I am resolving this conversation.

subtitle: "Researching the optimal locations to restore the urban streams in order to improve the biodiversity, climate adaptivity and quality of life in the city of Dresden"
author:
  - name: "Daan Schlosser"
    affiliation: "MSc Geomatics"
    role: "Project coordinator & data-analyst"
  - name: "Joost Bastiaansen"
    role: "Mapping specialist"
    affiliation: "MSc Urbanism"
  - name: "Aparnaa Chandrasekaran"
    role: "Research lead"
    affiliation: "MSc Urbanism"
  - name: "Teun van Dijk"
    affiliation: "MSc Urbanism"
    role: "Presentation & design lead"

format:
  html:
    includes:
      in-header: header.html
editor: visual
---

| Name                   | Affiliation   | Role                               |
|----------------------|------------------|--------------------------------|
| Daan Schlosser         | MSc Geomatics | Project coordinator & data-analyst |
| Joost Bastiaansen      | MSc Urbanism  | Mapping specialist                 |
| Aparnaa Chandrasekaran | MSc Urbanism  | Research lead                      |
| Teun van Dijk          | MSc Urbanism  | Presentation & design lead         |

<br>

## Introduction

### Context

> The European Union has started a project in 2024 focusing on the funding of Urban Stream Restoration, aiming to promote Biodiversity, Climate adaptation and to improve quality of life in cities (Interreg Central Europe, 2025). This program derives from the concept of Urban Stream Syndrome, which describes the consistently observed ecological degradation of watercourses that drain urban areas, where peak discharge rates during flood events are significantly higher, while baseflow is reduced due to decreased infiltration (Walsh et al., 2005). The cause for this Urban Stream Syndrome is the increase in impermeable surfaces, such as roads, rooftops, and parking lots, combined with the more efficient drainage of stormwater through pipelines. The combination of increased hard surfaces and efficient drainage results in a "flashier" hydrograph, which leads to more frequent and larger flow events, with faster rising and falling water flows. This causes certain symptoms of the Urban Stream Syndrome, starting with increased bank erosion and more channel incisions. Other effects are elevated concentrations and loads of chemical pollutants and nutrients in urban stream waters. Both symptoms negatively impact biodiversity (Walsh et al., 2005). With climate change, the urgency to address the limitations of urban drainage systems is constantly increasing (Arnbjerg-Nielsen et al., 2013). Impact analyses of urban runoff consistently show that, in regions impacted by climate change, a systematic adaptation effort is necessary to minimise the effects on the poor performance of current drainage systems, particularly in high-risk areas such as cities located along rivers (Arnbjerg-Nielsen et al., 2013). Some studies suggest that, despite the significance of climate change, systematically adapting urban streams can mitigate the impacts of urban stream syndrome to an acceptable level, which is one of the main goals of the Urban Stream Restoration project.

Reviewer: Note that ReBioClim (nowhere named in this introduction) is an Interreg CE project funded by the EU, not an EU initiative per se. Some more info here: https://www.interreg-central.eu/projects/rebioclim/

Reviewer: The introduction should also contain a paragraph with the main objective(s), the research question(s) and outline of the report.

## Methods

### Site Selection

This project will focus on the city of Dresden. In order to provide an answer to the main research question “what areas of Dresden would stand to gain the most from the urban stream restoration project?” a division of areas has to be provided. To start, a grid with tiles of 500 by 500 metres was defined for the city of Dresden. This is filtered to only show the tiles which contain a stream to remove any unnecessary information.

```{r}
#| echo: false

> library(slickR)

Reviewer: I am not sure if the caruousel is a good way to show your results for several reasons:

- You shouw the original analysis by default whereas in the results section I would expect to see the result.
- When the caurousel continues endlessly in both directions it is difficult to understand sequences: this is the case with the three-step process you describe here.
- Comparison is not possible because two images are never shown at the same time.

I suggest the less interactive but clearer alternative of showing the images side by side or one on top of each other in all cases where you used the carousel.

Reviewer 2: Agree, I'll remove the carousels and just use simple side by side images. These carousels were a pain to implement anyway and they still don't have the figure numbers and title.

library(htmltools)

intro_imgs <- list(
  'Step 1: A 500 by 500 metres grid covering Dresden' = "images/introduction/STEP1.png",
  'Step 2: Overlay with streams' = "images/introduction/STEP2.png",
  'Step 3: Remove any unnecessary information' = "images/introduction/STEP3.png"
)
htmltools::div(
  style = "max-width:100%; max-height:550px; overflow:hidden; margin:auto; position:relative;",
  slickR(obj = lapply(names(intro_imgs), function(name) {
    tags$figure(
      tags$img(
        src = intro_imgs[[name]],
        style = "max-width:100%; max-height:500px; height:auto; display:block; margin:auto;"
      ),
      tags$figcaption(name)
    )
  }), slideId = 'intro')
)
```

### MCDA

### Typology construction

> 

Reviewer: Describe the two methods here. _How_ did you carry out the analyses? What methodological decisions did you make? This does not have to be long but all steps you made have to be clear to the reader. This is closely related to reproducibility.

> ## Results

Reviewer: I would expect an overview before you start the three separate sub-sections and a conclusion or synthesis of the results at the end or the section.

### Quality of Life

> Apart from natural aspects such as biodiversity and climate adaptation, urban streams also exert an influence on the social level. The presence of an urban stream can directly influence the quality of life for the surrounding inhabitants of the area through creating a sense of community (Riley, 2016) or it is able to improve their mental health (Vert et al., 2020).

Reviewer: From how this paragraph is written, I would expect the sections on biodiversity and climate adaptation to be shown before it.

#### Vicinity to Urban Green Space

> x

Reviewer: Add missing descriptions.

  "Accessibility: Local Integration (Gridded)" = "images/quality_of_life/LocalIntegration_Grid.jpeg"
)
htmltools::div(
  style = "max-width:100%; max-height:550px; overflow:hidden; margin:auto; position:relative;",
  slickR(obj = lapply(names(qol_imgs), function(name) {
    tags$figure(
      tags$img(
        src = qol_imgs[[name]],
        style = "max-width:100%; max-height:500px; height:auto; display:block; margin:auto;"
      ),
      tags$figcaption(name)
    )
  }), slideId = 'Local_Integration')
)
```

#### Appearance

```{r}
#| echo: false
library(slickR)
library(htmltools)

qol_imgs <- list(
  "Accessibility: Appearance" = "images/quality_of_life/Covered.jpeg",
  "Accessibility: Appearance (Gridded)" = "images/quality_of_life/Covered_Grid.jpeg"
)
htmltools::div(
  style = "max-width:100%; max-height:550px; overflow:hidden; margin:auto; position:relative;",
  slickR(obj = lapply(names(qol_imgs), function(name) {
    tags$figure(
      tags$img(
        src = qol_imgs[[name]],
        style = "max-width:100%; max-height:500px; height:auto; display:block; margin:auto;"
      ),
      tags$figcaption(name)
    )
  }), slideId = 'Appearance')
)
```

#### Saaty Matrix

> ![Saaty Matrix Placeholder](images/quality_of_life/placeholder.jpg)

Reviewer: This comes a bit out of the blue. We need some context both in the methods section and here. Why are we using it? What does it show?

  slickR(obj = lapply(names(climate_imgs), function(name) {
    tags$figure(
      tags$img(
        src = climate_imgs[[name]],
        style = "max-width:100%; max-height:500px; height:auto; display:block; margin:auto;"
      ),
      tags$figcaption(name)
    )
  }), slideId = 'Seepage')
)
```

#### Infiltration Properties: Sealing Class

```{r}
#| echo: false
library(slickR)
library(htmltools)

climate_imgs <- list(
  "Infiltration: Sealing Class" = "images/climate_adaptation/6.1 Sealing_ Analysis.jpg",
  "Infiltration: Sealing Class (MCDA Grid)" = "images/climate_adaptation/6.2 Sealing_ MCDA Grid.jpg"
)
htmltools::div(
  style = "max-width:100%; max-height:550px; overflow:hidden; margin:auto; position:relative;",
  slickR(obj = lapply(names(climate_imgs), function(name) {
    tags$figure(
      tags$img(
        src = climate_imgs[[name]],
        style = "max-width:100%; max-height:500px; height:auto; display:block; margin:auto;"
      ),
      tags$figcaption(name)
    )
  }), slideId = 'Sealing_Class')
)
```

#### Saaty Matrix

![Saaty Matrix](images/climate_adaptation/Climate%20Adaptation_%20Saaty%20Matrix.jpg)

### Biodiversity

> Biodiversity is important because...

Reviewer: These introductory sentences are not about the importance or relevance of the overall objectives (that should be made clear in the introduction), but about site conditions and how those lead to the different results detaild in subsequent sub-sections. This comment applies to Climate Adaptation and Quality of Life as well.

  slickR(obj = lapply(names(biodiversity_imgs), function(name) {
    tags$figure(
      tags$img(
        src = biodiversity_imgs[[name]],
        style = "max-width:100%; max-height:500px; height:auto; display:block; margin:auto;"
      ),
      tags$figcaption(name)
    )
  }), slideId = 'intro')
)
```

#### Blue Green Percentage

```{r}
#| echo: false
library(slickR)
library(htmltools)

biodiversity_imgs <- list(
  "Blue green percentage" = "images/biodiversity/BlueGreenPercentage.png",
  "Blue green percentage gridded" = "images/biodiversity/BlueGreenPercentage_Gridded.png"
)
htmltools::div(
  style = "max-width:100%; max-height:550px; overflow:hidden; margin:auto; position:relative;",
  slickR(obj = lapply(names(biodiversity_imgs), function(name) {
    tags$figure(
      tags$img(
        src = biodiversity_imgs[[name]],
        style = "max-width:100%; max-height:500px; height:auto; display:block; margin:auto;"
      ),
      tags$figcaption(name)
    )
  }), slideId = 'intro')
)
```

#### Saaty Matrix

![Saaty Matrix Placeholder](images/biodiversity/placeholder.jpg)

### Map Dashboard

> Still W.I.P. but if you open cmd, use 'python -m http.server' and navigate to the directory with the index.html in it, it works. Just need to figure out how to get it directly into our qmd file, haven't looked into that yet.

Reviewer: Great that you set up a dashboard! Make sure you explain the purpose of it. You can also describe further steps needed for its improvement in the discussion (we do not expect you to develop a refined dashboard).

Reviewer: Note that:
- python needs to be installed on the computer running that command and that it might have to be replaced with `python3 -m http.server` depending on one's set-up (as in my case)
- one needs to run `http://[::]:8000/interactive%20map/` after running the python command you indicate.

### Overview of the normal distribution of our layers

To display the distribution of the normalised values for all our layers used in the saaty matrices, we've made a plot which displays these.

<div style="width:100%; text-align:center;">
```{r echo=FALSE, warning=FALSE, message=FALSE, dpi=900}
#install.packages("sf")
#install.packages("ggplot2")
#install.packages("tidyr")
#install.packages("dplyr")

library(sf)
library(ggplot2)
library(tidyr)

> library(dplyr)

Reviewer: Normally you put any set-up code like this in a separate code chunk named `setup` at the beginning of the report, right after the YAML header.

Reviewer: The last three are part of the tidyverse, so you can also replace them with `library(tidyverse)` to shorten the code.

  scale_fill_manual(values = c("Biodiversity" = "#415240", "Quality of Life" = "#765586", "Climate" = "#fbc010")) +
  labs(
    title = "The distribution of all our map layers: individual + weighted per theme",
    x = "Normalised Value",
    y = "Density",
    fill = NULL # No legend title
  ) +
  theme_minimal() +
  theme(
  strip.text = element_text(size = 7),
  axis.text = element_text(size = 7),
  axis.title = element_text(size = 8),        # smaller axis titles
  legend.text = element_text(size = 7),       # smaller legend text
  legend.title = element_text(size = 7),      # smaller legend title (if any)
  plot.title = element_text(size = 9),        # smaller plot title
  legend.position = "bottom",
  axis.ticks.x = element_line(size = 0.2)
)

```
</div>

### K-means clustering and topology definition

<div style="width:100%; text-align:center;">
```{r echo=FALSE, warning=FALSE, message=FALSE}
#install.packages("sf")
#install.packages("dplyr")
#install.packages("cluster")
#install.packages("plotly")

library(sf) # for processing vector data
library(dplyr) # for selecting and transforming data
library(cluster) # for the silhouette method

> library(plotly) # for 3D plot of the variables

Reviewer: See my comment on set-up code above. Merge everything from here that is different in such a setup code chunk.

# Create color vector matching the order of clusters in the data
plot_colors <- cluster_colors[as.character(as.numeric(grids$cluster))]

# Plot using base R plotting
plot(st_geometry(grids), 
     col = plot_colors,
     main = "Spatial Pattern of Urban Stream Clusters", 
     border = NA)

# Add legend showing cluster colors and IDs
legend("topright", 
       legend = paste("Cluster", names(cluster_colors)), 
       fill = cluster_colors, 
       title = "Clusters",
       bty = "n",  # No box around legend
       cex = 0.8)  # Smaller text size

# ------------------------------------------------------------------------------
# Extra 3D plot using plotly with matching colors

# Create the 3D plot with the same color mapping
plot_ly(
  x = ~features$BIO_Weighted,
  y = ~features$QOL_Weighted,
  z = ~features$CLI_Weighted,
  color = ~grids$cluster,
  colors = cluster_colors,  # Use the same colors as the spatial plot
  type = "scatter3d",
  mode = "markers"
) %>%
  layout(
    title = "3D Cluster Plot",
    scene = list(
      xaxis = list(title = "BIO_Weighted"),
      yaxis = list(title = "QOL_Weighted"),
      zaxis = list(title = "CLI_Weighted")
    )
  )
```
</div>

> ### Weighted maps per theme, and overall map

Reviewer: It is not clear if this is related to the MCDA or the typology. Please clarify.

Note: legend is accidently wrong, colours are correct.
![Biodiversity index, weighted](images/weighted/BIO_Weighted.png)
![Quality of Life index, weighted](images/weighted/QOL_Weighted.png)
![Climate Adaptation index, weighted](images/weighted/CLI_Weighted.png)
![Overall suitability index, weighted](images/weighted/Overall_Weighted.png)

## Discussion

> 

Reviewer: Here you discuss broader implications, further research, limitations, or other aspects that are important but fall outside the scope of your report.

## Conclusion

> 

Reviewer: Here you answer your research question succintly. One should be able to read the introduction and the conclusion and understand your main findings.

> ## Bibliography

Reviewer: **Optionally**, you can use Zotero to generate the Bibliography as described here: https://quarto.org/docs/authoring/citations.html

---
title: "Urban Stream Restoration in Dresden"
subtitle: "Researching the optimal locations to restore the urban streams in order to improve the biodiversity, climate adaptivity and quality of life in the city of Dresden"

format:
  html:
    number-sections: true
    fig-cap-location: bottom
    toc: true
    toc-location: left
    lightbox: true
    css: styles.css
    theme: Yeti
---

| Name                   | Affiliation   | Role                               |
|----------------------|-----------------|---------------------------------|
| Daan Schlosser         | MSc Geomatics | Project coordinator & data-analyst |
| Joost Bastiaansen      | MSc Urbanism  | Mapping specialist                 |
| Aparnaa Chandrasekaran | MSc Urbanism  | Research lead                      |
| Teun van Dijk          | MSc Urbanism  | Presentation & design lead         |

<br>

## Introduction

### Context

> The Interreg Central Europe programme, a funding initiative of the European Union for regional development in conjunction with climate change and the digital transition, launched the ReBioClim project in 2024. This project focuses on funding urban stream restoration, aiming to promote biodiversity, support climate adaptation, and improve quality of life in cities (Interreg Central Europe, 2025). The programme stems from the concept of Urban Stream Syndrome, which describes the consistent ecological degradation of watercourses that drain urban areas. In such areas, peak discharge rates during flood events are significantly higher, while baseflow is reduced due to decreased infiltration (Walsh et al., 2005). This is caused by the proliferation of impermeable surfaces—such as roads, rooftops, and parking lots—combined with more efficient stormwater drainage via pipelines. The combination of increased hard surfaces and efficient drainage results in a "flashier" hydrograph, leading to more frequent and intense flow events with rapidly rising and falling water levels. This contributes to symptoms of Urban Stream Syndrome, including increased bank erosion and greater channel incision. Other consequences include elevated concentrations and loads of chemical pollutants and nutrients in urban streams, all of which negatively impact biodiversity (Walsh et al., 2005).

Reviewer: I made some factual changes. ReBioClim is a _project_ funded as part of the Interreg CE funding _programme_.
```suggestion
The ReBioClim project started in 2024 as part of the Interreg Central Europe funding programme of the European Union for regional and transnational development. This project focuses on urban stream restoration, aiming to promote biodiversity, support climate adaptation, and improve quality of life in cities (Interreg Central Europe, 2025). The project stems from the concept of Urban Stream Syndrome, which describes the consistent ecological degradation of watercourses that drain urban areas. In such areas, peak discharge rates during flood events are significantly higher, while baseflow is reduced due to decreased infiltration (Walsh et al., 2005). This is caused by the proliferation of impermeable surfaces—such as roads, rooftops, and parking lots—combined with more efficient stormwater drainage via pipelines. The combination of increased hard surfaces and efficient drainage results in a "flashier" hydrograph, leading to more frequent and intense flow events with rapidly rising and falling water levels. This contributes to symptoms of Urban Stream Syndrome, including increased bank erosion and greater channel incision. Other consequences include elevated concentrations and loads of chemical pollutants and nutrients in urban streams, all of which negatively impact biodiversity (Walsh et al., 2005).
```

-   Which areas around urban streams are most vulnerable to flooding due to climate change?
-   Which areas along urban streams are most urgent to enhance residents’ day-to-day quality of life?
-   Which stream corridors have the highest potential to support and enhance urban biodiversity?

## Methods

To answer the main research question, it is important to select appropriate methods that consider environmental, social, and economic factors (Convertino et al., 2012). Clear metrics are required to measure the impact of restoration activities at various stages of the project. These metrics are based on three key themes, aligned with the goals of the ReBioClim project: **climate adaptation, quality of life, and biodiversity**. A literature review forms the basis for identifying which data are relevant to address the sub-questions and the main research question. It also explains the significance of each theme and how they are interconnected within the datasets. The structure of the research is schematised below.

![MCDA structure](images/introduction/Dresden%20MCDA.png)

> ### Site Selection

Reviewer: ```suggestion
### Site Selection and Unit of Analysis
```

In order to calculate the normalized value, the following calculation was used:

```{=html}
<pre> 
round( 
  CASE 
    WHEN "length" <= 140 THEN 0 
    WHEN "length" > 140 AND "length" <= 2500 THEN ( "length" - 140 ) / (2500 - 140) 
    ELSE 1 
  END,
  3 
)
</pre>
```

::::: {layout-ncol="2"}
::: column
![Appearance: Analysis](images/quality_of_life/Covered.jpeg){fig-cap="Accessibility: Appearance"}
:::

::: column
![Appearance: MCDA Grid](images/quality_of_life/Covered_Grid.jpeg){fig-cap="Accessibility: Appearance (Gridded)"}
:::
:::::

#### Saaty Matrix

> ![](images/quality_of_life/Quality%20of%20Life_Saaty%20Matrix.png)

Reviewer: The Saaty matrices require textual explanation as well. Should not be long, 2-3 sentences max., but explain what it shows.

Reviewer 2: Added for all three

> #### Recreational Suitability

Reviewer: Text is missing from here, right?

:::

::: column
![Flood Risk: MCDA Grid](images/climate_adaptation/1.2%20Flood%20risk_%20MCDA%20Grid.jpg){fig-cap="Flood Risk: MCDA Grid"}
:::
:::::

#### Urban Heat Island Effect

Higher Urban Heat Island (UHI) effects in Dresden elevate stream temperatures, reduce dissolved oxygen, and stress aquatic ecosystems. Impervious surfaces exacerbate run-off and thermal pollution, degrading urban water quality and biodiversity. Mitigating UHI is essential for urban stream resilience and climate adaptation.

Mitigating the Urban Heat Island (UHI) effect around urban streams enhances cooling, biodiversity, and public health. Vegetated riparian zones reduce temperatures through shading and evapotranspiration. Integrating blue-green infrastructure is vital for climate resilience in Dresden, especially around these streams (Bowler et al., 2010).

::::: {layout-ncol="2"}
::: column
![Urban Heat Island: Analysis](images/climate_adaptation/2.1%20UHI_%20Analysis.jpg){fig-cap="Urban Heat Island: Analysis"}
:::

::: column
![Urban Heat Island: MCDA Grid](images/climate_adaptation/2.2%20UHI_%20MCDA%20Grid.jpg){fig-cap="Urban Heat Island: MCDA Grid"}
:::
:::::

> #### Green Cover: Green Spaces

Reviewer: Text?

Thus, the sealing classes were normalised from 1-5, and the unsealed areas were given a value of 1. The grid was overlaid, and the mean values were calculated to see which areas require immediate sealing alteration to restore urban streams.

::::: {layout-ncol="2"}
::: column
![Sealing Class: Analysis](images/climate_adaptation/6.1%20Sealing_%20Analysis.jpg){fig-cap="Sealing Class: Analysis"}
:::

::: column
![Sealing Class: MCDA Grid](images/climate_adaptation/6.2%20Sealing_%20MCDA%20Grid.jpg){fig-cap="Sealing Class: MCDA Grid"}
:::
:::::

#### Saaty Matrix

> ![](images/climate_adaptation/Climate%20Adaptation_Saaty%20Matrix.png)

Reviewer: Description?

Reviewer 2: Added

#### How to use the dashboard

Unfortunately, due to CORS restrictions, you cannot open the dashboard by simply opening the HTML file. Fortunately, setting up a temporary local server is quite straightforward:

1.  Prerequisite: Python must be installed on your computer.
2.  Open your command prompt/terminal and navigate to the project directory, e.g.: cd C:\Users\daans\Documents\GitHub\report-asa2025-groupe
3.  Run python -m http.server or python3 -m http.server in the same terminal window, depending on your Python installation. This starts a temporary local server hosted by the terminal.
4.  Open your web browser and navigate to localhost:8000/interactive%20map/

### Weighted maps per theme

![Biodiversity index, weighted](images/weighted/BIO_Weighted.png)

![Quality of Life index, weighted](images/weighted/QOL_Weighted.png)

![Climate Adaptation index, weighted](images/weighted/CLI_Weighted.png)

### Weighted overall map

> ![Overall suitability index, weighted](images/weighted/Overall_Weighted.png)

Reviewer: Make sure you add some textual description as well

Reviewer 2: Added

> ### K-means clustering and topology definition

Reviewer: I am missing the text in this section

The use of space syntax, especially within the Attraction Reach module, has its limitations that may lead to inconsistencies in the analysis. Since all addresses were used as origins, areas without addresses, such as the Dresdner Heide, will be outliers within this analysis. This results in skewed data and may account for the moderate scores in the weighted quality of life map.

#### Proposals based on the analysis

Utilising Multi-Criteria Decision Analysis (MCDA), fifteen attributes were normalised and assigned weighted values to form three overarching objectives: biodiversity, climate adaptation, and quality of life, each given equal significance in the typological framework. While establishing a shared baseline for all urban streams enables consistent comparative analysis, it is critical to account for the specific weight distribution of the fifteen attributes when addressing underperforming cases. For the twelve lowest-ranked streams, differentiated design strategies must reflect the relative influence of each attribute, allowing context-sensitive interventions that align with site-specific ecological and social needs.

> ## Further research

Reviewer: This should be part of the discussion; may be at the end of it

In this project, all the urban streams within the Dresden boundary were considered. Considering the scale of the analysis, it was difficult to account for more zoomed-in understandings, such as the width of the stream and the riparian zone around, which are important while analysing and recommending context-specific solutions. For the twelve cases, a detailed analysis with a grid of 100-by-100 metres can be further considered to include stream widths, riparian zone width, vegetation cover, surface permeability, and proximity to water bodies, which are essential for stream restoration. However, such fine resolution may lead to data redundancy and increase complexity in citywide applications. A 50–100 m grid balances detail and feasibility for assessing ecosystem service distribution and hydrological patterns across urban subcatchments (Roest et al., 2023).

Additionally, other variables (if publicly available), other methodologies and 'translations' into the calculated normalised values may be considered with other, perhaps more specific (political, urban, etc.) goals in mind.

> ### Workflow reproducibility

Reviewer: Can also be part of the discussion

Reviewer 2: Yes, but I've put it specifically in the further research since I think that what I describe here is outside of the scope of this research.

In addition, future work could focus specifically on the project’s adherence to FAIR principles and improving our analysis reproducibility from A-Z. Regarding workflow reproducibility, an important improvement could be in addressing and reducing/ removing the current reliance on QGIS for spatial analysis, since this is inherently a way of working which consists of unscripted operations and without version control. Potentially, the QGIS operations could be scripted either using PyQGIS or plain programming languages like Python or R. However, if full scripting is not feasible, numbered instructions with screenshots and explanations about parameter values used would already significantly improve reproducibility, a sort of ‘Reproduction Guide’.

## Conclusion

Based on the MCDA conducted, the following map highlights the twelve streams that have the greatest potential to contribute to the city’s integrated climate resilience through immediate urban restoration measures for enhancing biodiversity, mitigating flood risks, and improving quality of life.

> ![Twelve urban streams fit for restoration](images/weighted/Conclusion_%201.png)

Reviewer: These are not twelve streams but twelve grid cells, right?

Reviewer 2: Yes, fixed it.

> ## Bibliography

Reviewer: Are you sure all bibliography is included here?

Reviewer 2: Yea, its not, i'll make sure it is by tonight
